### PR TITLE
html and css generation in main

### PR DESCRIPTION
**Pull request by balkamile**
html and css generation in main

### CODE REVIEW

File: main/main.c [hunk 1]
```diff
@@ -13,26 +13,289 @@ int main(int argc, char *argv[]) {
...
+    addCSSProperty(&sitContent, "display", "none");
+    pushStyle(&css, &sitContent);
+
+    Style show;
+    initStyle(&show, "show");
+    addCSSProperty(&show, "display", "block");
+    addCSSProperty(&show, "padding-left", "24px");
+    pushStyle(&css, &show);
 
-    pushStyle(&css, &bodyStyles);
```
Comment (AlexShukel) [inline main/main.c]: Extract this logic in separate file "init-css.c"

File: modules/card-info/card-info.c [hunk 2]
```diff
@@ -36,6 +36,20 @@ CardInfo initCardInfo() {
     return cardInfo;
 }
 
+int inputCards() {
```
Comment (AlexShukel) [inline modules/card-info/card-info.c]: You can use `bool` data type from `stdbool.h`

File: main/main.c [hunk 3]
```diff
@@ -13,26 +13,289 @@ int main(int argc, char *argv[]) {
...
+
+        for(int i=0; i<cardInfo.circumstancesSize; ++i)
+        {
+            Node *h_circumPoint = pushChildrenNode(h_circumList, "li", "");
+            pushStringLiteral(h_circumPoint, cardInfo.circumstances[i]);
+        }
+
+        Node *h_gradient = pushChildrenNode(h_pageContainer, "div", "id=\"gradient\"");
+
+        Node *h_script = pushChildrenNode(body, "script", "src=\"./index.js\"");
```
Comment (AlexShukel) [inline main/main.c]: Script node should be pushed only once after while loop. Generate index.js file separately too.

File: main/main.c [hunk 4]
```diff
@@ -13,26 +13,289 @@ int main(int argc, char *argv[]) {
...
+        Node *h_circumContent = pushChildrenNode(h_circumstances, "div", "class=\"sit-content\" id=\"situation-content\"");
+        Node *h_circumList = pushChildrenNode(h_circumContent, "ol", "class=\"normal-text\"");
+
+        for(int i=0; i<cardInfo.circumstancesSize; ++i)
+        {
+            Node *h_circumPoint = pushChildrenNode(h_circumList, "li", "");
+            pushStringLiteral(h_circumPoint, cardInfo.circumstances[i]);
+        }
+
+        Node *h_gradient = pushChildrenNode(h_pageContainer, "div", "id=\"gradient\"");
```
Comment (AlexShukel) [inline main/main.c]: Extract this logic in another file "card-html.c"

File: main/main.c [hunk 5]
```diff
@@ -13,26 +13,289 @@ int main(int argc, char *argv[]) {
...
+    pushChildrenNode(head, "link", "rel=\"preconnect\" href=\"https://fonts.gstatic.com\" crossorigin");
+    pushChildrenNode(head, "link", "href=\"https://fonts.googleapis.com/css?family=Poppins\" rel=\"stylesheet\"");
+
+    //Body
+    Node *body = htmlNode->children[1];
+
+    Node *h_pageContainer = pushChildrenNode(body, "div", "class=\"page-container\"");
+
+    Node *h_mainTitle = pushChildrenNode(h_pageContainer, "div", "class=\"title-text main-title\"");
+    pushStringLiteral(h_mainTitle, "UFO List - find\<br/\>your friends");
```
Comment (AlexShukel) [inline main/main.c]: Why do you use "\\<" and "\\>"?

**Review by AlexShukel — COMMENTED** [review]
Some comments.

File: main/init_css.c [hunk 6]
```diff
@@ -0,0 +1,184 @@
+#include "css-generator.h"
+#include "html-generator.h"
+#include "init-css.h"
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+void initCSS(CSS *css) {
```
Comment (AlexShukel) [inline main/init_css.c]: Function name collision in `css-generator.h`. Rename this function to `createCSS`. Rename files to `create-css.c` and `create-css.h` too.

File: include/init-css/init-css.h [hunk 7]
```diff
@@ -0,0 +1,13 @@
+#ifndef INIT_CSS_H
+#define INIT_CSS_H
+
+#include <stdio.h>
+#include <stdlib.h>
+
+/**
+ * Creates CSS file
```
Comment (AlexShukel) [inline include/init-css/init-css.h]: This function does not create CSS file. It creates and adds styles in CSS structure.

File: include/create-card/create-card.h [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+
+#include <stdio.h>
+#include <stdlib.h>
+
+/**
+ * Creates new card
+ * 
+ * @param h_cards Node in which the new card is pushed
+*/
+void createCard(Node *h_cards);
```
Comment (AlexShukel) [inline include/create-card/create-card.h]: Do not use underscores in names. We use `camelCase` convention.

**Review by AlexShukel — COMMENTED** [review]
Some comments.

**Review by Maksasj — APPROVED** [review]
gud !

**Pull request merged**